python/tvm/testing/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin, wildcard-import
+"""Utility Python functions for TVM testing"""
+from utils import *
```

Comment:
Should use 

```suggestion
from utils import *
```

BTW, do we really want wildcard import?

Reply:
I was just trying to get the subpackage to work on CI. Will avoid using wild card matching.

Reply:
One caveat of wildcard import is that it ignores names starting with underscore. We have [a few places](https://github.com/apache/tvm/blob/main/conftest.py#L35) outside the package that these names are used, wildcard here will not work

Reply:
done